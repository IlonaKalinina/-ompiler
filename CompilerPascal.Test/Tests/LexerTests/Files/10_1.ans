1 1 integer 1 1
2 7 integer 2 2
4 1 integer 3 3
End File