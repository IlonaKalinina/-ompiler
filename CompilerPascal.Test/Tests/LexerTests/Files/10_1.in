1 {comm
ents} 2		{comments
}
3{comments}