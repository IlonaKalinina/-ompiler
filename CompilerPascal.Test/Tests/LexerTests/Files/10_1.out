1 1 INT 1 1
2 7 INT 2 2
4 1 INT 3 3
